hydra-node/src/Hydra/Chain/Direct/State.hs

```diff
@@ -338,6 +340,14 @@ commit ctx st utxo = do
     (_, TxOut ShelleyAddressInEra{} _ _ _) ->
       Right ()
 
+  -- NOTE: Remove this limit once we have more experiments on mainnet.
+  rejectMoreThan100ADA :: NetworkId -> TxOut CtxUTxO -> Either (PostTxError Tx) ()
+  rejectMoreThan100ADA Mainnet output =
+    if selectLovelace (txOutValue output) > 100_000_000
+      then Left $ ReachedMainnetHardcodedLimit
+      else return ()
+  rejectMoreThan100ADA _network _txOut = return ()
```

Comment:
what is the convention here?
- return ()
- pure ()
- Right ()
 
imo using Right is better. wdyt?